docs/specs/dreamview_usage_table.md

```diff
@@ -10,9 +10,11 @@ Dreamview is a web application that,
 The application layout is divided into several regions: header, sidebar, main view, and tool view.
 
 ### Header
-The Header has 4 drop-downs that can be set as shown:
+The Header has 3 drop-downs that can be set as shown:
 ![](images/dreamview_usage_table/header.png) 
 
+The Co-Driver switch is used to detect disengagement event automatically. Once detect, dreamview will pop up the data recorder window for co-driver to enter a new drive event.
```

Comment:
The Co-Driver switch is used to the detect disengagement event automatically. Once detected, Dreamview will display a pop-up of the data recorder window for the co-driver to enter a new drive event.

Reply:
Changed. Thanks for the correction.